build/Dockerfile

```diff
@@ -1,7 +1,5 @@
-FROM alpine:3.8
-
-RUN apk upgrade --update --no-cache
+FROM fedora-minimal:30
```

Comment:
this should be `FROM registry.fedoraproject.org/fedora-minimal:30` to ensure the image is pulled from the proper location.

Reply:
correct, thanks